cypress/integration/executions_spec.js

```diff
@@ -69,5 +69,59 @@ describe('Dashboard Executions Page', () => {
         });
       });
     });
+
+    it('should show a single execution', () => {
+      cy.contains('nav li a', 'Executions').as('executions');
+      cy.get('@executions').should('have.attr', 'href', '#/executions');
+      cy.get('@executions').click();
+
+      cy.url().should('include', 'executions');
+      cy.contains('.heading--xlarge', 'Executions');
+      const executionName = '50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const executionArn = 'arn:aws:states:us-east-1:596205514787:execution:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf:50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const stateMachine = 'arn:aws:states:us-east-1:596205514787:stateMachine:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf';
+      cy.get('table tbody tr td[class=table__main-asset]').within(() => {
+        cy.get(`a[title=${executionName}]`).click({force: true});
+      });
+
+      cy.contains('.heading--large', 'Execution');
+      cy.contains('.heading--medium', 'Visual workflow');
+
+      cy.get('.table--wrapper')
+        .within(() => {
+          let i;
+          for (i = 26; i < 32; i++) {
```

Comment:
I'm still not quite sure I understand why we're loading a fixture with ~32 events if we're only going to test against 5 of them. Can we just reduce the fixture to 5ish events and test against all of them? 
That way we can avoid "magic" numbers in our testing

Reply:
On the other side, we want to test if dashboard can display complex workflow execution.  Probably only `HelloWorld` workflow execution has only 5 events.  It would be interesting to see how long it will take to verify all the events.

Reply:
If possible, I think we need to reduce the size of the fixture being loaded by the page/Cypress. At least on my machine, loading in this much data makes the test runner quite sluggish and less useful.

Reply:
I thought we didn't want to change the data on the page. And there is no execution that only has a few events. Are you sure that's what you want to have happen?